mk/rabbitmq-run.mk

```diff
@@ -165,9 +168,13 @@ define test_rabbitmq_config
 %% vim:ft=erlang:
 
 [
+  {lager, [
+      {error_logger_hwm, 5000}
+    ]},
   {rabbit, [
 $(if $(RABBITMQ_NODE_PORT),      {listeners$(COMMA) [$(RABBITMQ_NODE_PORT)]}$(COMMA),)
-      {loopback_users, []}
+      {loopback_users, []},
+      {log, [{file, [{level, debug}]}]}
```

Comment:
I find it useful when using `gmake run-broker` because this is difficult to configure otherwise, especially if you want to see debug boot messages.

I had this change locally for some time and thought it might be useful to others. Obviously, I can revert it if you believe this is wrong.

Reply:
OK! I didn't realize it's `run-broker` only.